projects/ui/src/shim.cds-core.scss

```diff
@@ -14,6 +14,11 @@
   @return calc(var(#{$variable}, var(#{$fallback_variable})) + (calc(4 / 20 * 1rem)));
 }
 
+// Reevaluate pixel values based on baseline rem setting.
+@function baselinePx($pixels: 0) {
+  @return calc($pixels * (1rem / var(--clr-global-font-size)));
+}
+
```

Comment:
As we agreed, this is a bad practice that should generally be avoided. 
I'll leave it in the PR because it has low risk, the shim will be removed soon and it will save a few review cycles at a moment close to the release.